dev-requirements-py2.txt

```diff
@@ -4,5 +4,7 @@ PyYAML==5.1.2
 pykwalify==1.7.0
 demisto-py==2.0.5
 pytest==4.6.5
+pytest-mock==1.11.1
 beautifulsoup4==4.8.1
-pypdf2==1.26.0
\ No newline at end of file
+pypdf2==1.26.0
+git+https://github.com/coyo8/parinx.git@6493798ceba8089345d970f71be4a896eb6b081d
```

Comment:
why don't we need this also in py3?

Reply:
It is used by docs script only which is py2. Not sure this package even supports py3.